packages/terra-docs/tests/wdio/button-spec.js

```diff
@@ -0,0 +1,27 @@
+Terra.describeViewports('Button', ['medium'], () => {
+  describe('Default Button', () => {
+    it('checks accessibility', () => {
+      browser.url('/raw/tests/terra-docs/docs/button');
+      browser.moveToObject('#root', 700, 700);
+      Terra.validates.element('checks accessibility-default', { selector: '#site' });
```

Comment:
This checks accessibility & takes a screenshot? should this just be title'd default?